src/wms/src/main/java/org/geoserver/wms/featureinfo/FreeMarkerTemplateManager.java

```diff
@@ -172,8 +170,7 @@ public FreeMarkerTemplateManager(
     }
 
     /** Writes the features to the output */
-    public boolean write(
-            FeatureCollectionType results, GetFeatureInfoRequest request, OutputStream out)
```

Comment:
If a backport is desired, the public API changes should be managed differently:
* Create a new parallel method
* Deprecate, but leave available, the old method
* Make sure nothing in GeoServer uses the old method

Repeat for the other public methods modified in this class.

Reply:
Done